tests/pipeline/test_execution_stage.py

```diff
@@ -0,0 +1,98 @@
+from datetime import datetime
+
+import pytest
+
+from snuba.attribution import get_app_id
+from snuba.attribution.attribution_info import AttributionInfo
+from snuba.clickhouse.columns import ColumnSet
+from snuba.clickhouse.query import Query
+from snuba.datasets.storages.storage_key import StorageKey
+from snuba.pipeline.query_pipeline import QueryPipelineResult
+from snuba.pipeline.stages.query_execution import ExecutionStage
+from snuba.query import SelectedExpression
+from snuba.query.data_source.simple import Table
+from snuba.query.dsl import and_cond, binary_condition, column, equals, literal
+from snuba.query.expressions import Column, FunctionCall
+from snuba.query.query_settings import HTTPQuerySettings
+from snuba.querylog.query_metadata import SnubaQueryMetadata
+from snuba.request import Request
+from snuba.utils.metrics.timer import Timer
+from snuba.utils.schemas import DateTime, UInt
+
+
+@pytest.mark.clickhouse_db
+@pytest.mark.redis_db
+def test_basic():
```

Comment:
the execution stage has a crucial step of writing to the querylog. We should make sure that happens in this test

Reply:
Also missing a test for dry run queries

Reply:
> it looks like writing to the querylog happens outside of the execution stage

ah okay that's good 😌 

still should add the dry run test though